test/types/request.test-d.ts

```diff
@@ -64,6 +77,10 @@ const getHandler: RouteHandler = function (request, _reply) {
   expectType<FastifyInstance>(request.server)
 }
 
+const getHandlerWithCustomLogger: RouteHandler = function (request, _reply) {
+  expectType<FastifyLoggerInstance>(request.log)
```

Comment:
Should not it be `CustomLoggerInterface`?

Reply:
Yes totally, I adjusted it thanks.